documentation/book/assembly-hello-world.adoc

```diff
@@ -0,0 +1,19 @@
+// This assembly is included in the following assemblies:
+//
+// getting-started.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {context}
+
+[id='hello-world-{context}']
+== Hello world example
+
+You can try this Hello world example to ensure everything is working fine for you.
```

Comment:
Please check for technical accuracy and change it to " This Hello World example provides instructions to deploy a client and communicate with the Kafka cluster. This example consists of producer and consumer which are communicating via Kafka cluster."